frontend/app/src/components/accounts/blockchain/ChainSelect.vue

```diff
@@ -46,51 +45,72 @@ const chains: SupportedChain[] = [
   }
 ];
 
-defineProps({
-  blockchain: {
-    required: true,
-    type: String as PropType<Blockchain>
-  },
-  disabled: {
-    required: true,
-    type: Boolean
+const props = withDefaults(
+  defineProps<{
+    modelValue?: Blockchain | null;
```

Comment:
it doesn't have to be `modelValue`,

`modelValue` is the placeholder you can replace with anything 

so for blockchain the pattern would be `blockchain` for the value and `update:blockchain` for the emit as it was before. 

and for vue3 this can then be used as a v-model -> `v-model:blockchain`

Reply:
but since this component sole purpose is to input blockchain (no other thing to bind), can't we use the default name `modelValue` instead `blockchain`, so later we just use `v-model` instead of `v-model:blockchain`